.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+
+defaults: &defaults
+  docker:
+    - image: docker.io/stackrox/apollo-ci:0.2.3
+      auth:
+        username: $DOCKER_IO_PULL_USERNAME
+        password: $DOCKER_IO_PULL_PASSWORD
+  working_directory: /home/circleci/go/src/github.com/stackrox/k8s-istio-cve-pusher
+
+
+setupGoogleAppCreds: &setupGoogleAppCreds
+  run:
+    name: Setup GCloud Service Account
+    command: |
+      touch /tmp/gcp.json
+      chmod 0644 /tmp/gcp.json
```

Comment:
So it doesn't make sense to set `0644` here. Setting `0600` is IMHO the better practice but also optional because we're running in an effectively single-user container. The suggestion was to skip `touch` and `chmod` and just use `>` for brevity - the file then having permissions `0644` is a tolerable side-effect, not a desired outcome.

Reply:
Reverting it back to chmod 0600. As discussed, @joshdk will fix this when he changes this job to run as k8s cron job.